feature/aft/aft_base/otg_tests/aft_base/README.md

```diff
@@ -0,0 +1,140 @@
+# AFT-1.1: AFTs Base
+
+## Summary
+
+IPv4/IPv6 unicast routes next hop group and next hop.
+
+## Procedure
+
```

Comment:
Since you have tests here, declare a subtest heading like:

```
## AFT-1.1.1: AFT ipv4/6 unicast entries
```

Reply:
updated